app/Http/Requests/UpdateUserRequest.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Validation\Rule;
+use Illuminate\Foundation\Http\FormRequest;
+
+class UpdateUserRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
```

Comment:
if you are calling the authrize inside your constructor controller or inside your func eg. `$this->authorize('update', auth()->user());` there no need to have it inside your request so remove this function `public function authorize()` if you won't use it